linters/tslint.json

```diff
@@ -0,0 +1,154 @@
+{
+  "extends": ["tslint-eslint-rules"],
+  "rules": {
```

Comment:
consider adding the `member-ordering` rule

here's what we use in our projects:
"member-ordering": [
    true,
    "public-before-private",
    "static-before-instance",
    "variables-before-functions"
]